WordPress/src/main/res/values/strings.xml

```diff
@@ -31,6 +31,7 @@
     <!-- Screen titles -->
     <string name="release_notes_screen_title">Release notes</string>
     <string name="help_screen_title">Help and Support</string>
```

Comment:
This is not in use anymore right?  Then it should be removed.

Reply:
Removed: e1f1d73a54abf855e7f4de45414f98916a64d4b5